.github/workflows/ci.yml

```diff
@@ -167,7 +167,11 @@ jobs:
         shell: bash
         run: |
           shopt -s extglob globstar nullglob
-          echo "$INPUTS_CHANGES" | xargs clang-format --dry-run --Werror
+          IFS=$'\n'
+          for FILE in $INPUTS_CHANGES; do
+              echo -n "$FILE" |
+              xargs -0 clang-format-12 --dry-run --Werror --verbose
+          done
```

Comment:
This handles any glob formats also takes care of any paths or globs that result in paths with spaces.